### PR TITLE
Removed one of the scrollbars and refactored the code

### PR DESCRIPTION
**Pull request by yasht01**
Removed the inner scrollbar and refactored some code by removing some spaces.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,7 +1,17 @@
+html {
+    height: 100%;
+    overflow-y: visible;
+}
+
```
Comment (yasht01) [inline css/style.css:5]: Added to show the scrollbar for the whole HTML window.

File: css/style.css [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
 #navbar {
     transition: 0.4s;
 }
 
+.divall {
+    overflow-x: hidden;
+    height: 100%;
+}
+
```
Comment (yasht01) [inline css/style.css:14]: Removed the scrollbar as it acted only on the single class and not on the complete window.

File: index.html [hunk 3]
```diff
@@ -48,8 +44,8 @@
                    
                 </ul>
 
-                <button class="btn btn-outline-warning my-2 my-sm-0" id="loginbtn" type="button" data-toggle="modal" data-target="#loginModal">Login</button>
-                <button class="btn btn-outline-warning my-2 my-sm-0" id="signupbtn" type="button" data-toggle="modal" data-target="#signModal">Sign Up</button>
```
Comment (yasht01) [inline index.html:52]: Added some spacing between the *Login* and *Signup* buttons.

File: index.html [hunk 4]
```diff
@@ -62,7 +58,7 @@
     </nav>
 
     <!-- Scrollspy -->
-    <div class="divall" data-spy="scroll" data-target="#navspy-example2" style=" overflow-x:hidden; height:1300px;">
```
Comment (yasht01) [inline index.html:65]: Moved these styles to the *.css* file.

**Review by yasht01 — COMMENTED** [review]
Please review.

**Comment by sagar-barapatre** [comment]
Great !!

**Pull request merged**